ghcide/bench/example/HLS

```diff
@@ -1 +0,0 @@
-../../..
```

Comment:
could you restore this file?

Reply:
Restored. I'm not sure how this went missing.

Thanks for all the feedback.